### PR TITLE
New blog post: Data Trail + R4DS solutions

### PR DESCRIPTION
**Pull request by mine-cetinkaya-rundel**
WIP!

### CODE REVIEW

**Comment by tracykteal** [comment]
Ready for review!

File: content/blog/data-trail/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+```
+Last year at rstudio::conf(2022) Jeff Leek [shared about the Data Trail program](https://youtu.be/Vf301YCxP1Q).
+
+> DataTrail is a no-cost, paid 14-week educational initiative for young-adult, high school and GED-graduates. DataTrail aims to equip members of underserved communities with the necessary skills and support required to work in the booming field of data science. DataTrail is a fresh take on workforce development that focuses on training both Black, Indigenous, and other people of color (BIPOC) interested in the data science industry ***and*** their potential employers.
```
Comment (hadley) [inline content/blog/data-trail/index.Rmd:37]: What clause does "and their potential employers" apply to?

**Review by hadley — APPROVED** [review]

File: content/blog/data-trail/index.Rmd (hunk 1 above)
Comment (mine-cetinkaya-rundel) [reply content/blog/data-trail/index.Rmd:37]: I'm not sure, but this is a direct quote from https://www.datatrail.org/.

File: content/blog/data-trail/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+Jabir's primary project was to work on the [R for Data Science solutions](https://mine-cetinkaya-rundel.github.io/r4ds-solutions/). The R for Data Science, 2nd Edition was released in June 2023. In this edition there is a lot of new content, and revisions and additions to exercises. We saw that [Jeffrey Arnold's solutions to the 1st edition](https://jrnold.github.io/r4ds-exercise-solutions/) are such a useful resource for the community. Therefore, this project aimed to both create a similar resource for 2nd edition and serve as an educational resource for the interns to help sharpen their tidyverse and general data science skills.
```
[the remark this answers is not in the document]
Comment (mine-cetinkaya-rundel) [reply content/blog/data-trail/index.Rmd:41]: 1e solutions are already linked, I opened an issue to link the 2e solutions as well, tracking at https://github.com/hadley/r4ds/issues/1555. I'll do it next time I'm in there.

File: content/blog/data-trail/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+ggplot(presidential, aes(x = start, y = id, color = party)) +
+  geom_point() +
+  geom_segment(aes(xend = end, yend = id)) +
+  scale_color_manual(values = c(Republican = "#E81B23", Democratic = "#00AEF3")) +
+  scale_x_date(
+    name = "Term",
+    breaks = seq(from = ymd("1953-01-20"), to = ymd("2021-01-20"), by = "4 years"),
+    date_labels = "'%y"
+  ) +
+  scale_y_continuous(
```
[the remark this answers is not in the document]
Comment (mine-cetinkaya-rundel) [reply content/blog/data-trail/index.Rmd:92]: Ah, I see, fancy 💅 . I agree this goes with the "Labelling each term with the name of the president." instruction.

I'll update to 

```r
ggplot(presidential, aes(x = start, y = id)) +
  geom_point(aes(color = party)) +
  geom_segment(aes(xend = end, yend = id, color = party)) +
  geom_text(aes(label = name), hjust = 0, vjust = 0, nudge_y = 0.1) +
  scale_color_manual(values = c(Republican = "#E81B23", Democratic = "#00AEF3")) +
  scale_x_date(
    name = "Term",
    breaks = seq(from = ymd("1953-01-20"), to = ymd("2021-01-20"), by = "4 years"),
    date_labels = "'%y"
  ) +
  scale_y_continuous(
    name = "President",
    breaks = 34:45
  ) +
  theme(
    panel.grid.minor = element_blank(),
    axis.ticks.y = element_blank()
  ) +
  theme(
    panel.grid.minor = element_blank(),
    axis.ticks.y = element_blank()
  ) +
  labs(
    x = "Term",
    y = "President",
    title = "Terms of US Presidents",
    subtitle = "Eisenhower (34th) to Trump (45th)",
    color = "Party"
  )
```

to capture goodies from both versions.

image

**Pull request merged**